### ISSUE TITLE
No image ready received from camera to MW4

### ISSUE THREAD

**Issue by ACMBakx** [mworion/MountWizzard4#301]
This is a new installation on another computer.
DellOptilex Win11Pro
First run after installation.

In contradiction to a former problem, I now get no image from the Touptek camera within MW4.
From NINA the image does come in. But not directly from within MW4
The camera connects alright, but when an image is taken the "image ready" message is not received.
MW4 waits indefinitely.
See log.
This happened in 01b. So I migrated to 3.0.0
Same thing.
The log from yesterday Feb 01 shows about images being taken via NINA but MW4 not able to have astap platesolve them.
This never happened to me before.
I had the mount connected to NINA as well to get the coordinates attached to the images.
Within NINA these images are solved by astap no problem.
I leave the  settings for image taking and astap solving at default.
That always has worked well. Surprisingly not anymore.    Computers . . . . . . 

Last week I was using a new installation on a Mele miniPC.
There it was working as we discussed before. But that computer has been replaced by the Dell again. 
This machine is  clearly more powerful and a lot faster with a i5  than the Celeron processor in the Mele.
Now I did a similar (identical?) installation on my Dell computer again. And the problem arises.

[mw4-2023-02-02.log](https://github.com/mworion/MountWizzard4/files/10569633/mw4-2023-02-02.log)
[mw4-2023-02-01.txt](https://github.com/mworion/MountWizzard4/files/10569665/mw4-2023-02-01.txt)

The second log file has a txt extension in a attempt to send it as attachment by mail.
But your new email service bounces mails with attachments. Also never happened before.
That is why I revert to this medium again.

Maybe you can shed some light on the matter.
Arie

**Comment by mworion** [comment]
Hi Arie,
this channel is the right one. All others I need to follow up individually. 
I checked the first log. Beside the fact as you remarked that image ready does does always responds False, the camera state is all the time on 2 which in ASCOM means on exposing. And this you see also on the release version.
That's strange. Would you mid to make a test with binning set to 2 or 4 ? For all the functions inside MW4 you only need about 1500 - 2000 pixels each axis.
Quick thoughts: 32 / 64 bit mixup. Is the ToupTek driver a 32 bit one?

For the second: If you check the FITS header of the NINA Images you would see RA = 0 DEC = 0 entry. So you don't have any hint for solving. NINA might transfer this based on the knowledge of the coordinates. So if it's a new install, please tell NINA to write the coordinates in the header. That's the source for MW4.

Michel

**Comment by mworion** [comment]
Arie,
as a user, would you mind if MW4 increases automatically the bin on large sensors to avoid this load on ascom interfaces ?
Michel

**Comment by ACMBakx** [comment]
I think ASTAP does a similar thing.
The binning only did not help.
It was the subframe setting at 50% that did the trick.
But if MW4 can do its magic with a little trick, I do not mind.

**Comment by mworion** [comment]
Hi Arie,
I add a hint for good binning, but I do not set it! See image:

![2023-02-24-16-13-55-MountWizzard4_-_v3 0 1a4](https://user-images.githubusercontent.com/6632980/221229610-2ec7e410-6593-4660-b019-165e66bd9227.png)

The value is to get 1750 pixels each axis.
Michel

**Comment by ACMBakx** [comment]
Thank you Michael.Makes sense.The cause of my recent problem with astap not solving was that I had not set the focal length at the page where it is needed.On a mount related page one sets the focal length and apperture, but also at a calculations page this value has to be entered. I have done a clean installation and this was the first run. I (finally) noticed in the tabel next to the presented image, that the pixel ratio was way of.So I spent nearly half an hour to locate the field where the focal length needs to be put. After that, with binning, the image was solved as astap uses to. In a blink an eye.Settings. Mount.  Telescope Information readoutImaging Statistics and Formulae. Imaging Information readout and calculationIt was the second one that I had forgot.Does it make sense to couple them?RegardsarieVerstuurd vanaf mijn iPadOp 24 feb. 2023 om 17:15 heeft Michael ***@***.***> het volgende geschreven:﻿
Hi Arie,
I add a hint for good binning, but I do not set it! See image:

The value is to get 1750 pixels each axis.
Michel
